src/shared/components/Badge/Badge.tsx

```diff
@@ -0,0 +1,21 @@
+import * as styles from './Badge.css';
+
+interface Props extends React.HTMLProps<HTMLDivElement> {
+  type: 'Hot' | '단독판매';
+}
```

Comment:
만약에 recipes로 변경하시면 여기는 

```suggestion
interface Props {
  type: 'Hot' | '단독판매';
}
```

이렇게만 작성하셔도 될 거 같아요 ㅎㅎ

Reply:
적용 완료!! (❁´◡`❁)